tests/integration/test_definitions/postgresql/integration_tests.py

```diff
@@ -220,9 +220,19 @@
     )
 ]
 
+fluent_datasources = [
+    IntegrationTestFixture(
+        name="how_to_connect_to_postgresql_data",
```

Comment:
@TrangPham Where is the actual module `tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/how_to_connect_to_postgresql_data.py`?  Thanks.